setup.py

```diff
@@ -1,11 +1,13 @@
 from setuptools import setup, find_packages
 
+
 base_packages = []
 
-test_packages = [
+test_packages = base_packages + [
     "pytest>=5.4.3",
     "black>=19.10b0",
     "flake8>=3.8.3",
+    "pytest-datafiles>=2.0",
```

Comment:
Is there a reason why you need this? Reading in a file could also be done with a `pytest.fixture` here. If we can omit dependencies that would be preferable.

Reply:
You're right. I could instead create a function that returns the name of the file